mlx_engine/vision/vision_model_wrapper.py

```diff
@@ -210,7 +289,7 @@ def _convert_to_pil(self, images_b64: List[str]):
             PIL.Image.open(BytesIO(base64.b64decode(img))) for img in images_b64 or []
         ]
 
-    def _custom_resize(self, pil_images, max_size=(1000, 1000)):
+    def _custom_resize(self, pil_images, max_size=(512, 512)):
```

Comment:
When I initially was testing to set this to 1000x1000, if I dropped it down to 512 models like qwen2vl would start mis-recognizing text from screenshots of my screen. 

Why the change? If for certain models can we custom resize just for those models?

Reply:
I didn't mean to commit this change. I reverted this.